src/Test/Unit/Config/SchemaTest.php

```diff
@@ -169,7 +169,7 @@ public function testGetDefaultsForGlobalSection(): void
                 StageConfigInterface::VAR_DEPLOYED_MAGENTO_VERSION_FROM_GIT => '',
                 StageConfigInterface::VAR_DEPLOY_FROM_GIT_OPTIONS => [],
                 StageConfigInterface::VAR_MIN_LOGGING_LEVEL => '',
-                StageConfigInterface::VAR_X_FRAME_CONFIGURATION => 'SAMEORIGIN',
+                StageConfigInterface::VAR_X_FRAME_CONFIGURATION => 'SAMEORIGIN'
```

Comment:
What the point of such changes? ) As I remember you always was a supporter of coma after the last array element.

Reply:
Reverted, thanks